### DISCUSSION TITLE
Remove POI-s from map

### DISCUSSION THREAD

**Discussion by bkekelic2** [mapsforge/mapsforge#1268] [Q&A] [answered]
Hi,
I integrated mapsforge with osmdroid in Android application. 
The map works great 👍, but my question is if there is any way to remove some details from the map.
I downloaded .map files from your [mirror links](http://ftp-stud.hs-esslingen.de/pub/Mirrors/download.mapsforge.org/maps/v5/europe/) and it is quite detailed with some details I don't even need, like POI-s, some street labels etc. Also, I should be able to zoom quite to see streets and buildings so using lower zoom is not an option.

Is there any way to remove such a details maybe through _MapsForgeTileProvider_, _osmdroid_ interface or directly from the .map file?
Thanks in advance.

**Comment by devemux86** [comment, marked as answer]
You can use / create a render theme with map layers menu, see the [render theme documentation](https://github.com/mapsforge/mapsforge/blob/master/docs/Rendertheme.md#stylemenus).

You can try [OpenAndroMaps](https://www.openandromaps.org/en) maps with Elevate map style.

**Reply by bkekelic2** [reply]
Thanks for your suggestions. 
I picked this [theme ](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-themes/src/main/resources/assets/mapsforge/osmarender.xml) and commented rules I don't need to hide different icons on the map and it worked. 
Thanks again.
